lvm/clean.sls

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+## Automate tasks in logical sequence ...
+
+include:
+    - lvm.lv.clean
+    - lvm.vg.clean
+    - lvm.pv.clean
+    - lvm.profiles.clean
+    - lvm.files.clean
```

Comment:
Shouldn't we use relative import?

Reply:
yes, I'll raise an issue for the comment when I can.

Reply:
Ok, so I resolve this discussion.